mbofui/app/views/popup.html

```diff
@@ -12,6 +12,13 @@
     <small>{{event.address}}</small>
   </div>
   <hr class="popupRule">
+  <p ng-show="event.guests !==0">
+    <span class="badge badge-default">{{event.guests}} </span> have signed up.
+  </p>
+  <p ng-show="event.guests ===0">
+    No one has signed up yet.
```

Comment:
If I'm an owner of an event and I see this an hour before my event starts I may be inclined to sob a little.